connect/runtime/src/test/java/org/apache/kafka/connect/util/KafkaBasedLogTest.java

```diff
@@ -147,10 +168,6 @@ public void setUp() {
 
     @Test
     public void testStartStop() throws Exception {
-        expectStart();
-        expectStop();
-
-        PowerMock.replayAll();
 
```

Comment:
Nit: can we remove all empty lines at the beginning and ends of methods?

Reply:
Done.